lib/internal/v8_prof_polyfill.js

```diff
@@ -96,6 +96,13 @@ function readline() {
     if (line.length === 0) {
       return '';
     }
+    if (bytes === 0) {
+      process.emitWarning('profile file is broken', {
```

Comment:
Suggestion: you might just add the filename of the broken file? And please use a capital letter as first character.

Reply:
Thank you for the advice.Add the filename is better.